README.md

```diff
@@ -1 +1,59 @@
-# blog-app
\ No newline at end of file
+<!-- @format -->
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Blog App
+
+>"Blog App" is a classic example of a blog website. A fully functional website that will show the list of posts and empower readers to interact with them by adding comments and liking posts.
+
+## Built With
+
+- Ruby 
+- Rails
+- Postgres
+
+## Getting Started
+
+### To launch the project locally:
+
+#### Step 1:
+
+- Clone this repo locally using git in the command line and the following command
+
+ ```bash
+ > git clone git@github.com:zhadier/blog-app.git
+ ```
+
+- Alternatively, you can just download the complete zip file and extract the folder in your directory
+
+#### Step 2:
+
+In the project directory, you can run:
+
+```bash
+> rails s
+```
```

Comment:
- [ ] I would recommend adding the proper steps to clone and run the project locally to make the README professional.

```
$ git clone https://github.com/zhadier/blog-app.git
$ cd blog-app
$ bundle
$ rails db:setup
$ rails s
```